code/modules/jobs/job_types/fishdoctor.dm

```diff
@@ -0,0 +1,55 @@
+/datum/job/fishdoctor
+	title = JOB_FISHDOCTOR
+	description = "Catch fish, breed fish, eat fish."
```

Comment:
![bild](https://user-images.githubusercontent.com/17747087/167305748-d0361c1b-3fc8-4ef7-907b-30b903aab19e.png)
perfectly good opportunity and you didnt do it

Reply:
Maybe put it on the examine more tag.